src/test/java/com/myhome/controllers/unit/AmenityControllerTest.java

```diff
@@ -154,9 +162,46 @@ void shouldNotAddAmenityWhenCommunityNotExists() {
     assertEquals(HttpStatus.NOT_FOUND, response.getStatusCode());
   }
 
+  @Test
+  void shouldReturnListOfBookingsOfGivenAmenityId() {
```

Comment:
Can you apply given/when/then pattern to tests?

Reply:
done